Makefile

```diff
@@ -21,28 +21,25 @@ SUBDIR+=	hatponboard-lib
 SUBDIR+=	head-action
 SUBDIR+=	hey5-description
 SUBDIR+=	hpp-affordance
-SUBDIR+=	hpp-benchmark
-SUBDIR+=	hrp2-14-description
-SUBDIR+=	hrp2-dev
-SUBDIR+=	infuse-asn1-conversions
-SUBDIR+=	infuse-asn1-types
-SUBDIR+=	infuse-envire
-SUBDIR+=	infuse-idl
-SUBDIR+=	infuse-msgs-ros
-SUBDIR+=	infuse-novatel-gps-driver-ros
+#SUBDIR+=	infuse-asn1-conversions
```

Comment:
Please don't comment the packages you are not using: they will be used on the buildfarm